jerry-core/ecma/base/ecma-globals.h

```diff
@@ -294,8 +294,8 @@ typedef enum
  */
 typedef struct
 {
-  mem_cpointer_t getter_p : ECMA_POINTER_FIELD_WIDTH; /**< pointer to getter object */
-  mem_cpointer_t setter_p : ECMA_POINTER_FIELD_WIDTH; /**< pointer to setter object */
+  __extension__ mem_cpointer_t getter_p : ECMA_POINTER_FIELD_WIDTH; /**< pointer to getter object */
+  __extension__ mem_cpointer_t setter_p : ECMA_POINTER_FIELD_WIDTH; /**< pointer to setter object */
```

Comment:
The following _could_ work somewhere in jrt.h where the attributes are defined.
```c
#ifndef __GNUC__
#define __extension__
#endif
```
But do note that right now we have no compiler specific ifdefs in the core code base.